api/HTMLSelectElement.json

```diff
@@ -317,7 +317,7 @@
               "version_added": true
             },
             "opera_android": {
-              "version_added": null
+              "version_added": true
```

Comment:
https://github.com/mdn/browser-compat-data/pull/3732 sets `setCustomValidity` support to true, can be confirmed by `HTMLSelectElement.prototype.setCustomValidity` or `document.createElement('select').setCustomValidity` manually.

Don't know what's gone wrong here, it's in the Chromium's IDL:
https://cs.chromium.org/chromium/src/third_party/blink/renderer/core/html/forms/html_select_element.idl?l=56&rcl=aae10192b9f96a766448a4e48570d67c775b9d63

Reply:
For one, this is a comment on `checkValidity`.  It looks like `setCustomValidity` was initially set to `false` when migrating from the MDN tables, and was simply never fixed.

Reply:
Had to map from conflicts locally to the right lines in the review and failed here. I'll comment again in the right place.